homeassistant/components/modbus/const.py

```diff
@@ -0,0 +1,72 @@
+"""Constants used in modbus integration."""
+
+# configuration names
+CONF_BAUDRATE = "baudrate"
+CONF_BYTESIZE = "bytesize"
+CONF_HUB = "hub"
+CONF_PARITY = "parity"
+CONF_STOPBITS = "stopbits"
+CONF_REGISTER = "register"
+CONF_REGISTER_TYPE = "register_type"
+CONF_REGISTERS = "registers"
+CONF_REVERSE_ORDER = "reverse_order"
+CONF_SCALE = "scale"
+CONF_COUNT = "count"
+CONF_PRECISION = "precision"
+CONF_OFFSET = "offset"
+
+# integration names
+DEFAULT_HUB = "default"
+MODBUS_DOMAIN = "modbus"
+
+# data types
+DATA_TYPE_CUSTOM = "custom"
+DATA_TYPE_FLOAT = "float"
+DATA_TYPE_INT = "int"
+DATA_TYPE_UINT = "uint"
+
+# register types
+CALL_TYPE_COIL = "coil"
+CALL_TYPE_COILS = "coils"
```

Comment:
This is actually an array, not an attribute. It's equivalent to the `CONF_INPUTS` field below. Consider renaming to `CONF_COILS`.

Here's an actual example:
```yaml
switch:
  platform: modbus
  coils:
    - name: plc_01_door_manual
      hub: plc_01
      slave: 1
      coil: 1
...
```